docs/en/query_language/functions/date_time_functions.md

```diff
@@ -62,9 +62,42 @@ Converts a date with time to a UInt8 number containing the number of the minute
 Converts a date with time to a UInt8 number containing the number of the second in the minute (0-59).
 Leap seconds are not accounted for.
 
-## toUnixTimestamp
+## toUnixTimestamp {#to_unix_timestamp}
 
-Converts a date with time to a unix timestamp.
+Converts a date with time to a Unix timestamp.
+
+**Syntax** 
+
+```sql
+toUnixTimestamp(datetime[, timezone])
+```
+
+**Parameters**
+
+- `datetime` — Date with time. [DateTime](../../data_types/datetime.md#data_type-datetime).
+- `timezone` — Optional time zone argument. [String](../../data_types/string.md).
```

Comment:
Timezone parameter does not make sense if DateTime value is passed in first parameter.

Reply:
Fix it